dev/ci/ci-perennial.sh

```diff
@@ -6,7 +6,4 @@ ci_dir="$(dirname "$0")"
 FORCE_GIT=1
 git_download perennial
 
-# required by Perennial's coqc.py build wrapper
-export LC_ALL=C.UTF-8
-
-( cd "${CI_BUILD_DIR}/perennial" && git submodule update --init --recursive && make TIMED=false )
+( cd "${CI_BUILD_DIR}/perennial" && git submodule update --init --recursive && make TIMED=false -j "${NJOBS}" lite )
```

Comment:
```suggestion
( cd "${CI_BUILD_DIR}/perennial" && git submodule update --init --recursive && make TIMED=false lite )
```
We have magic in the ci-common sourced above to handle njobs

Reply:
I'm committing the change and merging myself to shorten time with broken ci

Reply:
Awesome, thanks!